backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,414 @@
+an event to which the sample mixer can react (i.e. stop mixing the silence...).
+
+"Mute all" doesn't generate anything, it basically just lowers the volume of the
+music to zero.
+
+"No music" means using the null audio plugin which prevents generating any MIDI
+music (and therefore avoiding the expensive synthesis emulation) but beware, it
+doesn't affect CD (*.wav) playback at all!
+
+So for the best performance, always choose "No music" in the GUI options when
+the game contains MIDI tracks and "Mute" when the game contains a sampled
+soundtrack.
+
+Please note that it is not that bad, you surely can play The Secret of Monkey
+Island with AdLib enabled (but the CD/talkie versions sound better and
+are cheaper to play ;)).
+
+Vsync in GUI
+~~~~~~~~~~~~
+
+Carefully with the vsync option. It can easily cripple direct/single buffer
+rendering by 10-15 FPS if not used with caution. That happens if a game takes,
+say, 1.2 frames per update (so causing screen tearing anyway and rendering the
+option useless) but Vsync() forces it to wait 2 full frames instead.
+
+By the way, the vsync flag in Global Options affects also the overlay rendering
+(with all the pitfalls which apply to the single buffering mode)
+
+Slow GUI
+~~~~~~~~
+
+Themes handling is quite slow - each theme must be depacked, each one contains
+quite a few XML files to parse and quite a few images to load/convert. Best is
+to use the built-in one. A compromise solution is to depack the theme in an
+equally named directory. It will be loaded, too but without the depacking
+overhead.
+
+
+Known issues
+------------
+
+- aspect ratio correction works on RGB only (yet)
+
+- SuperVidel's DVI output is stretched when in 320x200 or 640x400; I'll  wait
+  for other people's experiences, maybe only my LCD is so lame.
+
+- adding a game in TOS and loading it in FreeMiNT (and vice versa) generates
+  incompatible paths. Either use only one system or edit scummvm.ini and set
+  there only relative paths (mintlib bug/limitation).
+
+- the default theme ("ScummVM Modern Theme Remastered") is not loaded
+  automatically on plain 8+3 filesystem. If you want to use it nevertheless,
+  choose it again in the Misc/Theme option (but you can't have the theme
+  depacked as a directory; many xml/image files use long filenames).
+
+- the talkie version of MI1 needs to be merged from two sources: first generate
+  the DOS version and then additionally also the flac version. Then convert all
+  *.flac files into *.wav and replace monkey.sof (flac) with monster.sou (DOS).
+  And of course, don't forget to set the extra path in Game options to the
+  folder where *.wav files are located! For MI2 just use the DOS version,
+  there are no CD tracks available. :(
+
+
+Future plans
+------------
+
+- aspect ratio correction for VGA/SuperVidel
+
+- unified file paths in scummvm.ini
+
+- 8bpp overlay (and get rid of all that 16bpp GUI crap)
```

Comment:
We added support for RGB332 overlays recently, so you might want to give that a go.

Reply:
Oh, that is a very recent addition. Following the stream of changes I see that my (not yet 100% well working) implementation was pretty close to his, although I was following this advice: https://forums.scummvm.org/viewtopic.php?p=57964#p57964. Thanks for the tip!